lib/normalizr/concern.rb

```diff
@@ -10,7 +10,7 @@ def normalize(*args, &block)
 
         prepend Module.new {
           options.attributes.each do |method|
-            define_method :"#{method}=" do |value|
+            define_method :"#{method}=" do |value, *args|
```

Comment:
Could you please change the name of the `args` variable here so it doesn't conflict with the `args` passed to the `normalize` method?

Reply:
done